src/proxyClient.js

```diff
@@ -129,6 +129,11 @@ var SUPPORT_BASE64_EMBEDDING;
 var filename;
 filename ||= '<<< filename >>>';
 
+#if SINGLE_FILE && SINGLE_FILE_BINARY_ENCODE
+#include "binaryDecode.js"
+var workerURL = URL.createObjectURL(new Blob([binaryDecode(filename)], {type: 'application/javascript'}));
+var worker = new Worker(workerURL);
```

Comment:
Looks like this line is shared between the `if` and the `else` here.. can we move it below?